build.py

```diff
@@ -1102,17 +1102,17 @@ def create_dockerfile_linux(
 RUN python3 -m pip install --upgrade pip \\
       && pip3 install transformers
 
-# Uninstall unused nvidia packages
-RUN if pip freeze | grep -q "nvidia.*"; then \\
-        pip freeze | grep "nvidia.*" | xargs pip uninstall -y; \\
-    fi
-RUN pip cache purge
```

Comment:
I was  wondering why do we need to remove this section? I think this is just to reduce some bytes for the container size. Do we see issues if remove those libraries?

Reply:
Because it eventually crash the CI